setup.cfg

```diff
@@ -1,44 +0,0 @@
-[flake8]
```

Comment:
Was nothing using this content, with everything using [.flake8](https://github.com/bjlittle/iris/blob/migrate-to-cirrus-ci/.flake8) instead?

As I understand it, this PR has so far only _added_ to the tools running `flake8` (Stickler is still being used at the time of writing).

Reply:
Yup, this wasn't needed. The `flake8` configuration is in the `.flake8` file instead

Reply:
It appears I wrote this at the same time as @bjlittle removed Stickler (5dfd8a160f899bd0bfc2e7379fd3fba12db8bcc4). Does that explain this removal?

Reply:
Thanks